ffi/SDL2_0.lua

```diff
@@ -114,7 +114,11 @@ function S.waitForEvent(usecs)
 
 		-- if we got an event, examine it here and generate
 		-- events for koreader
-		if event.type == SDL.SDL_KEYDOWN then
+		if ffi.os == "OSX" and (event.type == SDL.SDL_FINGERMOTION or
+			event.type == SDL.SDL_FINGERDOWN or
```

Comment:
Btw, what's the problem with the FINGERDOWN event on Mac OS?

Reply:
Both FINGER inputs and emulated MOUSE inputs will be generated with Mac trackpad so we will just ignore the FINGER ones.